docs/golang/index.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Ethereum for Go Developers
+meta:
+  - name: description
+    content: Learn how to develop for Ethereum using Go-based projects and tooling
+  - property: og:title
+    content: Ethereum for Go Developers
+  - property: og:description
+    content: Learn how to develop for Ethereum using Go-based projects and tooling
+lang: en-US
+sidebar: auto
+sidebarDepth: 0
+---
+
+# Ethereum for Go Developers
+<div class="featured">Learn how to develop for Ethereum using Go-based projects and tooling</div><br>
+
+Use Ethereum to create decentralized applications (or "dapps") that utilize the benefits of cryptocurrency and blockchain technology. These dapps can be trustworthy, meaning that once they are deployed to Ethereum, they will always run as programmed. They can control digital assets in order to create new kinds of financial applications. They can be decentralized, meaning that no single entity or person controls them and are nearly impossible to censor.
+
+## Getting Started with Smart Contracts and the Solidity Language
+
+**Take your first steps to integrating Go with Ethereum**
+
+Need a more basic primer first? Check out [ethereum.org/learn](/learn/) or [ethereum.org/developers](/developers/).
+
+- [Blockchain Explained](https://kauri.io/article/d55684513211466da7f8cc03987607d5/blockchain-explained)
+- [Understanding Smart Contracts](https://kauri.io/article/e4f66c6079e74a4a9b532148d3158188/ethereum-101-part-5-the-smart-contract)
+- [Write your First Smart Contract](https://kauri.io/article/124b7db1d0cf4f47b414f8b13c9d66e2/remix-ide-your-first-smart-contract)
+- [Learn How to Compile and Deploy Solidity](https://kauri.io/article/973c5f54c4434bb1b0160cff8c695369/understanding-smart-contract-compilation-and-deployment)
+- [Contract Tutorial](https://github.com/ethereum/go-ethereum/wiki/Contract-Tutorial)
+
+## Beginner resources
+- [Choosing an Ethereum Client](https://www.trufflesuite.com/docs/truffle/reference/choosing-an-ethereum-client)
+- [Use Golang to Connect to Ethereum](https://www.youtube.com/watch?v=-7uChuO_VzM)
+- [Deploy Ethereum Smart Contracts Using Golang](https://www.youtube.com/watch?v=pytGqQmDslE)
+- [A Step By Step Guide To Testing and Deploying Ethereum Smart Contracts in Go](https://hackernoon.com/a-step-by-step-guide-to-testing-and-deploying-ethereum-smart-contracts-in-go-9fc34b178d78)
+- [Go Ethereum book](https://goethereumbook.org/)
+
+
+## Intermediate articles
+- [Official geth Documentation](https://nethereum.readthedocs.io/en/latest/ethereum-and-clients/geth/)
```

Comment:
Your link is to the nethereum doucmentation - I believe this is the official Geth documentation: https://geth.ethereum.org/docs/

Reply:
I will change it.
Thank you